### PR TITLE
Kate/87510/add existing Take Profit component to Turbos Trade params

### PR DESCRIPTION
**Pull request by kate-deriv**
## Changes:

- Add existing Rick Management component for responsive. It's visible only in non-collapsed (expanded) version (according to design).
- Add existing Take Profit component for responsive. It's shown only when "Rick Management" Trade param it tapped.

## What is negotiable:

- In the existing code we have 'collapsible' attribute for div tag, but this attribute do not exist in official documentation https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes. It's also not a custom attribute, as all of them should start with 'data-', so I have no clear understanding of how it works and why do we need it. If the value of this attribute will be changed from 'true' to 'false' nothing changed.
Example of usage:
```
<div collapsible='true'>
        <BarrierMobile />
</div>
```
- After clicking on info icon near 'Take profit' on mobile, we show a tooltip, but it goes beyond (cross the border) the cell phone screen. Same on production.
- Barrier component is visible only in non-collapsed (expanded) version, because on production for Trade types, which are using barriers (for e.g. 'Touch/ No Touch'), we also show it only in non-collapsed (expanded) version. This goes against the proposed design, so clarification is needed.

## When you need to add unit test

-   [ ] If this change disrupt current flow
-   [ ] If this change is adding new flow

## When you need to add integration test

-   [ ] If components from external libraries are being used to define the flow, e.g. @deriv/components
-   [ ] If it relies on a very specific set of props with no default behavior for the current component.

## Test coverage checklist (for reviewer)

-   [ ] Ensure utility / function has a test case 
-   [ ] Ensure all the tests are passing

## Type of change

-   [ ] Bug fix
-   [ ] New feature
-   [ ] Update feature
-   [x] Refactor code
-   [ ] Translation to code
-   [ ] Translation to crowdin
-   [ ] Script configuration
-   [ ] Improve performance
-   [ ] Style only
-   [ ] Dependency update
-   [ ] Documentation update
-   [ ] Release

### CODE REVIEW

File: packages/trader/src/Modules/Trading/Components/Form/screen-small.jsx [hunk 1]
```diff
@@ -66,6 +67,11 @@ const CollapsibleTradeParams = ({
                     <RiskManagementInfo />
                 </div>
             )}
+            {is_turbos && (
+                <div collapsible='true'>
+                    <RiskManagementInfo />
+                </div>
+            )}
```
Comment (maryia-deriv) [inline packages/trader/src/Modules/Trading/Components/Form/screen-small.jsx]:
```suggestion

```
@kate-deriv instead, i suggest adding `is_turbos &&` to line 65 like this 🙂:
{is_multiplier && is_turbos && (

Comment (kate-deriv) [reply packages/trader/src/Modules/Trading/Components/Form/screen-small.jsx]: Great idea, please excuse my inattention

Comment (kate-deriv) [reply packages/trader/src/Modules/Trading/Components/Form/screen-small.jsx]: But may be we should add {is_multiplier || is_turbos && ...} as they can't be both equal to true at the same time

Comment (maryia-deriv) [reply packages/trader/src/Modules/Trading/Components/Form/screen-small.jsx]: yeah, sorry, i meant `||` 😅

**Review by maryia-deriv — APPROVED** [review]
👍🏻

**Pull request merged**